src/components/IncidenceReportsComponent.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Table, Header, Pagination } from 'semantic-ui-react';
+
+import TableRowComponent from './TableRowComponent.jsx';
+import SideMenuComponent from '../_components/SideMenuComponent.jsx';
+
+import { loadIncidenceReports } from '../_actions/incidenceReports.actions';
+
+export class IncidenceReportsComponent extends React.Component {
+  state = {
+    activePage: 1,
+    limit: 10,
+  }
+
+  componentDidMount() {
+    this.props.loadIncidenceReports();
+  }
+
+  handlePaginationChange = (e, { activePage }) => {
+    this.setState({ activePage });
+    this.props.loadIncidenceReports(activePage);
+  }
+
+  handlePageTotal = () => Math.ceil(this.props.incidenceReportsCount / this.state.limit)
+
+  emptyReportsCheck = () => (this.props.reports.length === 0)
+
+  loadTableContent = () => {
+    if (this.emptyReportsCheck()) {
+      return <Table.Row><Table.Cell colSpan="8">No Data Found</Table.Cell></Table.Row>
+    } else {
+      const incidenceReports = this.props.reports.map((incidenceReport, index) => {
+        return (
+          <TableRowComponent
+            key={index}
+            data={incidenceReport}
+            headings={['asset',
+              'incident_type',
+              'incident_location',
+              'incident_description',
+              'injuries_sustained',
+              'loss_of_property',
+              'witnesses',
+              'police_abstract_obtained'
+            ]}
+          />
+        )});
+     return incidenceReports;
+    }
+  }
+
+  render() {
+    return (
+      <SideMenuComponent>
+        <Header className="landing-heading" content="Incidence Reports" />
+        <Table celled>
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell>Asset</Table.HeaderCell>
+              <Table.HeaderCell>Incident Type</Table.HeaderCell>
+              <Table.HeaderCell>Incident Location</Table.HeaderCell>
+              <Table.HeaderCell>Incident Description</Table.HeaderCell>
+              <Table.HeaderCell>Injuries Sustained</Table.HeaderCell>
+              <Table.HeaderCell>Loss of Property</Table.HeaderCell>
+              <Table.HeaderCell>Witnesses</Table.HeaderCell>
+              <Table.HeaderCell>Police Abstract</Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+
+          <Table.Body>
+            {
+              this.loadTableContent()
+            }
+          </Table.Body>
+
+          <Table.Footer>
+            <Table.Row>
+              <Table.HeaderCell colSpan="8">
+                {
+                  (this.emptyReportsCheck())
```

Comment:
A ternary operator whose one expression renders nothing to the UI can be simplified with `&&` to:
```
{!this.emptyReportsCheck() && (
  <Pagination
    totalPages={this.handlePageTotal()}
    onPageChange={this.handlePaginationChange}
    activePage={this.state.activePage}
  />
)}
```